versioned_docs/version-latest/operators_and_staking/operators.md

```diff
@@ -322,3 +323,77 @@ Only the account who registered **Operator** can swith the domain.
 Stake of your **Nominators** won't be released, but will be moved to the new domain as well. 
 :::
 
+
+### Useful commands
+#### Running both validator (farmer) and operator nodes at the same time
+
+:::tip
+To run both operator and validator at the same time, provide requrired flags for both roles when starting your node. 
+:::
+
+<Tabs groupId="OS">
+<TabItem value="windows" label="🖼️ Windows" default>
+
+```powershell
+target/production/subspace-node `
+# validator flags
+--chain gemini-3g `
+--blocks-pruning 256 `
+--state-pruning archive `
+--no-private-ipv4 `
+--validator `
+--name your_node_name `
+# operator flags
+-- `
+--domain-id 1 `
+--operator `
+--keystore-path /tmp/keystore `
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp `
+```
+
+</TabItem>
+
+
+<TabItem value="macos" label="🍎 macOS">
+
+```bash
+target/production/subspace-node `
+# validator flags
+--chain gemini-3g \
+--blocks-pruning 256 \
+--state-pruning archive \
+--no-private-ipv4 \
+--validator \
+--name your_node_name \
+# operator flags
+-- \
+--domain-id 1 \
+--operator \
+--keystore-path /tmp/keystore \
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp `
```

Comment:
```suggestion
```

Reply:
Oops, didn't push the last commit which fixed this, my bad.